### PR TITLE
Master

### PR DESCRIPTION
**Pull request by ghost**
Please god let this work

Version for merging

### CODE REVIEW

**Comment by ghost** [comment]
derp derpington

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -18,4 +18,5 @@
         </activity>
     </application>
     <uses-permission android:name="android.permission.INTERNET"></uses-permission>
+
```
Comment (leec94) [inline app/src/main/AndroidManifest.xml:21]: LOOKS GR8

File: app/src/main/java/com/example/caroline/smashrunmobile/FirstPage.java [hunk 2]
```diff
@@ -65,42 +47,17 @@ public void doSomething(){
         String userid = userIdField.getText().toString();
 
         Log.d("SmashRunMobile", "User Id: " + userid);
-        TextView t = (TextView)findViewById(R.id.outTextView);
-
-        t.setText("test :] ");
-
-        configureJSON();
```
Comment (leec94) [inline app/src/main/java/com/example/caroline/smashrunmobile/FirstPage.java:72]: this looks fishy

File: app/src/main/java/com/example/caroline/smashrunmobile/FirstPage.java [hunk 3]
```diff
@@ -65,42 +47,17 @@ public void doSomething(){
...
-    }
 
-    public void configureJSON() {
```
Comment (leec94) [inline app/src/main/java/com/example/caroline/smashrunmobile/FirstPage.java:75]: change configureJSON back to original

**Pull request merged**